workers/unity/Assets/Playground/Scripts/UI/ConnectionScreenController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Improbable.Gdk.Core;
+using Improbable.Gdk.Mobile;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Playground
+{
+    public class ConnectionScreenController : MonoBehaviour
+    {
+        private const string CachedIp = "cachedIp";
```

Comment:
This string `cachedIp` for playerprefs needs to be more specific to improbable/GDK for unity

Also the variable name should reflect that it's used for player prefs